test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+	
+	@Test
+	public void nullObject_returnsTrue() {
+		Integer a = null;
+		assertTrue(Utils.isAnyNull(a));
+	}
+	
+	@Test
+	public void notNullObject_returnsFalse(){
+		Integer a = 1;
+		assertFalse(Utils.isAnyNull(a));
+	}
+	
+	@Test
+	public void UniqueElements_returnsTrue(){
+		ArrayList<String> array = new ArrayList<String>();
+		array.add("a");
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.